packages/core/src/servient.ts

```diff
@@ -48,12 +48,18 @@ export default class Servient {
         // initializing forms fields
         thing.forms = [];
         for (const name in thing.properties) {
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
```

Comment:
Is this intentional? If I understood it correctly, this would lead to the line not being transpiled at all.

Reply:
Oh, sorry, you are right, feel free to mark this as resolved :)